msteams-platform/graph-api/rsc/resource-specific-consent.md

```diff
@@ -148,7 +174,21 @@ The RSC permissions are declared in your app manifest JSON file. Add a [webAppli
 >
 > [!IMPORTANT]
 > Non-RSC permissions are stored in the Azure portal. Do not add them to the app manifest.
->
+
+### Example of RSC Permission for older version(Placeholder for version) and latest version(Placeholder for version)
+
+> [!NOTE]
+> For latest version(placeholder for version) the RSC permissions have two sections as RSC Delegated  permissions and RSC Application permission in Authorization section.
+
+The following are the links for RSC permissions for older version:
+
+* [Example for RSC in a team](#example-for-rsc-in-a-team)
+* [Example for RSC in a chat](#example-for-rsc-in-a-chat)
+
+The following are the links for RSC permissions for latest version:
+
+* [Example for RSC Delegated permissions](#example-for-rsc-delegated-permissions)
+* [Example for RSC Application permissions](#example-for-rsc-application-permissions)
 
 ### Example for RSC in a team
```

Comment:
This is what I was expecting.

### Example for requesting RSC permissions in a team when app manifest version is less than 1.x.
```json
"webApplicationInfo": {
    "id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
    "resource": "https://RscBasedStoreApp",
    "applicationPermissions": [
        "TeamSettings.Read.Group",
        "TeamSettings.ReadWrite.Group",
        "ChannelSettings.Read.Group",
        "ChannelSettings.ReadWrite.Group",
        "Channel.Create.Group",
        "Channel.Delete.Group",
        "ChannelMessage.Read.Group",
        "TeamsAppInstallation.Read.Group",
        "TeamsTab.Read.Group",
        "TeamsTab.Create.Group",
        "TeamsTab.ReadWrite.Group",
        "TeamsTab.Delete.Group",
        "TeamMember.Read.Group",
        "TeamsActivity.Send.Group"
    ]
  }
  ```
  
  
### Example for requesting RSC permissions in a team when app manifest version 1.x or greater.
```json
"webApplicationInfo": {
    "id": "XXxxXXXXX-XxXX-xXXX-XXxx-XXXXXXXxxxXX",
    "resource": "https://RscBasedStoreApp"
    },
"authorization": {
    "permissions": {
        "resourceSpecific": [
            {
                "name": "TeamSettings.Read.Group",
                "type": "Application"
            },
            {
                "name": "TeamSettings.ReadWrite.Group",
                "type": "Application"
            },
            {
                "name": "ChannelSettings.Read.Group",
                "type": "Application"
            },
            {
                "name": "ChannelSettings.ReadWrite.Group",
                "type": "Application"
            },
            {
                "name": "Channel.Create.Group",
                "type": "Application"
            },
            {
                "name": "Channel.Delete.Group",
                "type": "Application"
            },
            {
                "name": "ChannelMessage.Read.Group",
                "type": "Application"
            },
            {
                "name": "TeamsAppInstallation.Read.Group",
                "type": "Application"
            },
            {
                "name": "TeamsTab.Read.Group",
                "type": "Application"
            },
            {
                "name": "TeamsTab.Create.Group",
                "type": "Application"
            },
            {
                "name": "TeamsTab.ReadWrite.Group",
                "type": "Application"
            },
            {
                "name": "TeamsTab.Delete.Group",
                "type": "Application"
            },
            {
                "name": "TeamMember.Read.Group",
                "type": "Application"
            },
            {
                "name": "TeamsActivity.Send.Group",
                "type": "Application"
            }
        ]    
    }
}
  }
  ```

Reply:
And then I would expect you to apply the same example for chats.

Reply:
Resolved.